frontend/src/components/WeaponInvetory.vue

```diff
@@ -0,0 +1,727 @@
+  margin: auto;
+  display: block;
+
+  transform: scale(0.7);
+}
+
+.frame-0{
+    animation: none;
+}
+
+.frame-1{
+  border: 1px solid  rgba(0, 162, 255, 0.5) !important;
+}
+
+.frame-2{
+   border: 1px solid rgba(125, 0, 125, 0.5) !important;
+}
+
+.frame-3{
+   border: 1px solid rgba(255, 102, 0, 0.3) !important;
+}
+
+.frame-4{
+   border: 1px solid rgba(125, 0, 0, 0.5) !important;
+}
+
+.glow-0 {
+  animation: none;
+}
+
+.glow-1 {
+  box-shadow: inset 0 0 15px rgba(0, 162, 255, 0.5);
+}
+
+.glow-2 {
+  box-shadow: inset 0 0 20px rgba(125, 0, 125, 0.5);
+}
+
+.glow-3 {
+  box-shadow: inset 0 0 25px rgba(255, 102, 0, 0.3);
+}
+
+.glow-4 {
+  box-shadow: inset 0 0 30px rgba(125, 0, 0, 0.5);
+}
+
+
+.no-durability {
+  opacity: 0.6;
+}
+
+
+
+.weapon-flex{
+  display: flex;
+  flex-direction: row;
+}
+
+.weapon-img{
+  background-color: rgba(0, 0, 0, 0.213);
+  border-radius: 10px;
+  border: rgb(3, 141, 21) 1px solid;
+}
+
+.weapon-details{
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  padding: 10px 20px;
+  flex-grow: 1;
+}
+
+.weapon-img > img{
+  max-width: 100px;
+  min-width: 100px;
+  min-height: 100px;
+  max-height: 100px;
+}
+
+.trait-name{
+  color:#fff;
+  font-size: 11px;
+  padding-right: 10px;
+  border-right: 1px solid rgba(255, 255, 255, 0.261);
+}
+
+.name {
+  font-size: 17px;
+  color: #fff;
+  font-size: 0.9em;
+  text-align: left;
+  text-transform: capitalize !important;
+  font-weight: 800;
+}
+
+
+.id {
+  position: absolute;
+  top: 8px;
+  left: 10px;
+  font-style: normal;
+  font-size: 12px;
+  color: #fff;
+}
+
+
+.id, .trait, .stats {
+  /* position: absolute; */
+}
+
+.trait {
+  display: flex;
+  align-items: center;
+  /* top: 10px; */
+  /* left: 10px; */
+}
+
+.trait .traits{
+  margin-right: 10px;
+  /* padding-right: 10px; */
+  padding: 5px 0px;
+}
+
+.favorite-star {
+  /* position: absolute; */
+  /* margin-left: 110px; */
+}
+
+.stats {
+  display: flex;
+  align-items: center;
+  font-size: 11px;
+  /* top: 35px; */
+  /* left: 10px; */
+}
+
+.btn-equip{
+  border-radius: 3px !important;
+  padding: 2px 15px;
+  font-size: 11px;
+  background-color: rgba(255, 255, 255, 0.275);
+  width: fit-content;
+  color: #fff;
+  border: none !important;
+  margin-top: 10px;
+}
+
+
+.bonus-power {
+  /* position: absolute; */
+  /* bottom: 40px;
+  right: 10%; */
+  font-size: 0.6em;
+  text-align: right;
+}
+
+.small-durability-bar {
+  position: relative;
+  /* top: -5px; */
+  height: 3px;
+  width: 100%;
+  margin: 0 auto;
+  /* border-radius: 2px; */
+  background : linear-gradient(to right, rgb(236, 75, 75) var(--durabilityReady), rgba(255, 255, 255, 0.1) 0);
+}
+
+
+@media all and (max-width: 600px) {
+  .img-adventure > img{
+    max-width: 70px;
+    min-width: 70px;
+    min-height: 70px;
+    max-height: 70px;
+  }
+
+  .img-adventure > span{
+    position: absolute;
+    bottom:10px;
+    right:10px;
+  }
+
+  .img-adventure{
+    border-radius: 7px;
+  }
+}
+
```

Comment:
This is a monstrous file. There is definitely some code that could be cleaned up / removed. but I think this was a copy and paste from somewhere else?